esp/esp/themes/theme_data/bigpicture/less/variables.less

```diff
@@ -1,3 +1,2 @@
 @identityBlue: #1C1953;
-@highlightOrange: #FF8000;
-@pale: #f8f8f8;
\ No newline at end of file
+@highlightOrange: #FF8000;
```

Comment:
Can we rename these to something a little more descriptive? (then we won't need as much help text below)

Reply:
In hindsight, we might not be able to reduce the help text too much, but I think changing the names is still useful.